drivers/pwm/pwm_mcux_pwt.c

```diff
@@ -85,6 +86,20 @@ static int mcux_pwt_configure_capture(const struct device *dev,
 		return -EBUSY;
 	}
 
+#if defined(CONFIG_SOC_SERIES_KE1XZ)
+	if ((flags & PWM_CAPTURE_TYPE_MASK) == PWM_CAPTURE_TYPE_BOTH) {
+		LOG_ERR("Cannot capture both period and pulse width");
+		return -ENOTSUP;
+	}
+
+	if ((flags & PWM_CAPTURE_TYPE_MASK) == PWM_CAPTURE_TYPE_PERIOD) {
+		if ((flags & PWM_POLARITY_MASK) == PWM_POLARITY_NORMAL) {
```

Comment:
nit: instead of nesting conditionals, just use an &&

Reply:
Thanks for reminding, I will update it.